File: 5173登录/1.js
const crypto=require('crypto-js')
a='wpx5s1'
c='123'
function main_(a,c){
    return crypto.MD5(crypto.MD5(c).toString().substr(8, 16)+ a).toString()
}

console.log(main_(a, c));

// c = hex_md5(hex_md5(c).substr(8, 16) + a)

File: 5173登录/1.py
import hashlib
def hex_md5(s):
    m = hashlib.md5()
    m.update(str(s).encode("utf-8"))
    return m.hexdigest()
a='wpx5s1'
p='123'
p = hex_md5(hex_md5(p)[8:8 + 16] + a)
print(p)